README.md

```diff
@@ -1,185 +1,167 @@
-
-### Quick Summary
-
-IPFS is a protocol:
-- defines a content-addressed file system
-- coordinates content delivery
-- combines Kademlia + BitTorrent + Git
-
-IPFS is a filesystem:
-- has directories and files
-- mountable filesystem (via FUSE)
-
-IPFS is a web:
-- can be used to view documents like the web
-- files accessible via HTTP at `https://ipfs.io/<path>`
-- browsers or extensions can learn to use the `ipfs://` URL or `dweb:/ipfs/` URI schemes directly
-- hash-addressed content guarantees authenticity
-
-IPFS is modular:
-- connection layer over any network protocol
-- routing layer
-- uses a routing layer DHT (kademlia/coral)
-- uses a path-based naming service
-- uses bittorrent-inspired block exchange
+## Quick summary
 
-IPFS uses crypto:
-- cryptographic-hash content addressing
-- block-level deduplication
-- file integrity + versioning
-- filesystem-level encryption + signing support
+The IPFS project seeks to evolve the infrastructure of the Internet and the Web, with many things we've learned from successful systems, like [Git](https://git-scm.com/), [BitTorrent](http://bittorrent.org/), [Kademlia](https://en.wikipedia.org/wiki/Kademlia), [Bitcoin](https://bitcoin.org/), and many, many more. This is the sort of thing that would have come out of ARPA/DARPA, IETF, or Bell Labs in another age. IPFS is a free, open-source project with thousands of contributors.
 
-IPFS is p2p:
-- worldwide peer-to-peer file transfers
-- completely decentralized architecture
-- **no** central point of failure
+IPFS ([the InterPlanetary File System](https://docs-beta.ipfs.io/concepts/what-is-ipfs/)) is a hypermedia distribution **protocol** addressed by content and identities. It enables the creation of completely distributed applications, and in doing so aims to make the web faster, safer, and more open. 
 
-IPFS is a cdn:
-- add a file to the filesystem locally, and it's now available to the world
-- caching-friendly (content-hash naming)
-- bittorrent-based bandwidth distribution
+IPFS is a **distributed file system** that seeks to connect all computing devices with the same system of files. In some ways, this is similar to the original aims of the Web, but IPFS is actually more similar to a single BitTorrent swarm exchanging Git objects. You can read more about its origins in the paper [IPFS - Content Addressed, Versioned, P2P File System](https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf?raw=true).
 
-IPFS has a name service:
-- IPNS, an SFS inspired name system
-- global namespace based on PKI
-- serves to build trust chains
-- compatible with other NSes
-- can map DNS, .onion, .bit, etc to IPNS
+IPFS is becoming a **new major subsystem of the internet**. If built right, it could complement or replace HTTP. It could complement or replace even more. Let's go point-by-point into how.
```

Comment:
I'm going to scroll quickly through these and assume they are copy-paste from somewhere where this has been reviewed.

Reply:
All the material in this section was already in the old readme -- I just consolidated two places where we were essentially saying the same thing twice.